augly/image/__init__.py

```diff
@@ -73,6 +74,7 @@
     ApplyLambda,
     Blur,
     Brightness,
+    Skew,
```

Comment:
Please put this in alphabetical order in the list of transform classes (after `ShufflePixels`, before `VFlip`).